### PR TITLE
[zh-cn]: update the translation of the import statement

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

As the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#18266

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 1]
```diff
@@ -42,181 +42,187 @@ import "module-name";
...
-import("/modules/my-module.js").then((module) => {
-  // Do something with the module.
-});
+myModule.doAllTheAmazingThings();
 ```
 
-这种使用方式也支持 `await` 关键字。
+`myModule` is a [sealed](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/isSealed) object with [`null` prototype](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#null-prototype_objects). The default export available as a key called `default`. For more information, see [module namespace object](/en-US/docs/Web/JavaScript/Reference/Operators/import#module_namespace_object).
+
+`myModule` 是一个[密封](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/isSealed)的对象，其原型为 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)。它提供了一个名为 `default` 的键，用于访问默认导出。有关更多信息，请参考[模块命名空间对象](/zh-CN/docs/Web/JavaScript/Reference/Operators/import#模块命名空间对象)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:141]:
```suggestion
`myModule` 是一个[密封](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/isSealed)的对象，其原型为 [`null`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)。它提供了一个名为 `default` 的键，用于访问默认导出。有关更多信息，请参考[模块命名空间对象](/zh-CN/docs/Web/JavaScript/Reference/Operators/import#模块命名空间对象)。
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 title: import
 slug: Web/JavaScript/Reference/Statements/import
+l10n:
+  sourceCommit: 4f86aad2b0b66c0d2041354ec81400c574ab56ca
 ---
 
 {{jsSidebar("Statements")}}
 
-静态 **`import`** 语句用于导入由另一个模块导出的绑定。无论是否声明了 {{jsxref("Strict_mode","strict mode")}}，导入的模块都运行在严格模式下。在浏览器中，`import` 语句只能在声明了 `type="module"` 的 `script` 的标签中使用。
+静态 **`import`** 声明用于导入由另一个模块导出的只读实时{{Glossary("binding", "绑定")}}。导入的绑定被称为*实时绑定*，因为它们会由导出绑定的模块更新，但导入模块不能重新赋值。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:10]: 参考了[动态标准](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#html)，考虑一下哪个比较好

```suggestion
静态 **`import`** 声明用于导入由另一个模块导出的只读动态（live）{{Glossary("binding", "绑定")}}。导入的绑定被称为*动态绑定*，因为它们会由导出绑定的模块更新，但导入模块不能重新赋值。
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
-此外，还有一个类似函数的动态 `import()`，它不需要依赖 `type="module"` 的 script 标签。
+为了在源文件中使用 `import` 声明，该文件必须被运行时解释为一个[模块](/zh-CN/docs/Web/JavaScript/Guide/Modules)。在 HTML 中，这可以通过在 {{HTMLElement("script")}} 标签中添加 `type="module"` 来实现。模块会自动以[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)解释。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:12]:
```suggestion
要在源文件中使用 `import` 声明，该文件必须被运行时解释为[模块](/zh-CN/docs/Web/JavaScript/Guide/Modules)。在 HTML 中，这可以通过在 {{HTMLElement("script")}} 标签中添加 `type="module"` 来实现。模块会自动以[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)解释。
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 
-在 [script](/zh-CN/docs/Web/HTML/Element/script) 标签中使用 `nomodule` 属性，可以确保向后兼容。
-
-在你希望按照一定的条件或者按需加载模块的时候，动态 `import()` 是非常有用的。而静态型的 `import` 是初始化加载依赖项的最优选择，使用静态 `import` 更容易从代码静态分析工具和 [tree shaking](/zh-CN/docs/Glossary/Tree_shaking) 中受益。
+此外，还有一个类似函数的动态 [`import()`](/zh-CN/docs/Web/JavaScript/Reference/Operators/import)，它不需要依赖 `type="module"` 的 script 标签。
 
 ## 语法
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:17]: 语法的参数部分需要更新一下

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 5]
```diff
@@ -42,181 +42,185 @@ import "module-name";
 
 ## 描述
 
-`name` 参数是“导入模块对象”的名称，它将用一种名称空间来引用导入模块的接口。export 参数指定单个的命名导出，而 `import * as name` 语法导入所有导出接口，即导入模块整体。以下示例阐明该语法。
+`import` 声明只能出现在模块中，并且只能在顶级作用域（即不能在块、函数等内部）。如果在非模块上下文中遇到 `import` 声明（例如，没有 `type="module"` 的 `<script>` 标签、`eval`、`new Function`，这些都有"脚本"或"函数体"作为解析目标），则会抛出 `SyntaxError`。要在非模块上下文中加载模块，请改用[动态导入](/zh-CN/docs/Web/JavaScript/Reference/Operators/import)语法。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:45]:
```suggestion
`import` 声明只能出现在模块中，并且只能在顶级作用域中（即不能在块、函数等内部）。如果在非模块上下文中遇到 `import` 声明（例如，没有 `type="module"` 的 `<script>` 标签、`eval`、`new Function`，这些都有“脚本”或“函数体”作为解析目标），则会抛出 `SyntaxError`。要在非模块上下文中加载模块，请改用[动态导入](/zh-CN/docs/Web/JavaScript/Reference/Operators/import)语法。
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 6]
```diff
@@ -42,181 +42,185 @@ import "module-name";
...
-在这里，访问导出接口意味着使用模块名称（在本例为“myModule”）作为命名空间。例如，如果上面导入的模块包含一个接口 `doAllTheAmazingThings()`，你可以这样调用：
+`import` 声明有四种形式：
 
-```js
-myModule.doAllTheAmazingThings();
-```
+- [具名导入](#具名导入): `import { export1, export2 } from "module-name";`
+- [默认导入](#默认导入): `import defaultExport from "module-name";`
+- [命名空间导入](#命名空间导入): `import * as name from "module-name";`
+- [副作用导入](#仅为了其副作用而导入模块): `import "module-name";`
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:58]:
```suggestion
- [具名导入](#具名导入)：`import { export1, export2 } from "module-name";`
- [默认导入](#默认导入)：`import defaultExport from "module-name";`
- [命名空间导入](#命名空间导入)：`import * as name from "module-name";`
- [副作用导入](#仅为了其副作用而导入模块)：`import "module-name";`
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 7]
```diff
@@ -42,181 +42,185 @@ import "module-name";
...
-getUsefulContents("http://www.example.com", (data) => {
-  doSomethingUseful(data);
-});
+console.log(getPrimes(10)); // [2, 3, 5, 7]
 ```
 
-### 动态导入
+### 导入的值只能由导出者修改
+
+被导入的标识符是一个*实时绑定*，因为它可能由导出它的模块重新赋值。但是，导入它的模块不能重新赋值。即使导出了一个对象的模块也可能对其进行修改，并会影响到所有导入该值的模块。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:199]:
```suggestion
被导入的标识符是一个*动态绑定*，因为它可能由导出它的模块重新赋值。但是，导入它的模块不能重新赋值。然而，任何拥有导出对象的模块都可修改该对象，并会影响到所有导入该值的模块。
```

File: files/zh-cn/web/javascript/reference/statements/import/index.md [hunk 8]
```diff
@@ -42,181 +42,185 @@ import "module-name";
...
+import { myValue } from "/modules/my-module.js";
+import * as myModule from "/modules/my-module.js";
+
+console.log(myValue); // 1
+console.log(myModule.myValue); // 1
+setTimeout(() => {
+  console.log(myValue); // 2；my-module 更新了它的值
+  console.log(myModule.myValue); // 2
+  myValue = 3; // TypeError: Assignment to constant variable.
+  // 导入模块只能读取该值但不能重新分配它。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/statements/import/index.md:222]:
```suggestion
  // 导入模块只能读取该值但不能重新对它赋值。
```

**Review by yin1999 — COMMENTED** [review]
一些建议

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you @JasonLamv-t

**Pull request merged**